src/src_V2/components/sidebar/Sidebar.module.css

```diff
@@ -9,12 +9,16 @@
   width: 40px !important;
   height: 40px !important;
   top: 15px;
-  z-index: 1201;
+  z-index: 1;
```

Comment:
@aninaslyan jan why we need this change?

Reply:
z-index of Sidebar conflicted with the Drawer (for row edit) I created. Then I decided to change overlapping priorities (z-indexes) one by one with normal smaller numbers: Sidebar - 0, Sidebar's toggle button - 1. And other z-indexes can be added accordingly, not like z-index: 999999

Reply:
got it